modules/ROOT/pages/runtime-installation-task.adoc

```diff
@@ -45,6 +45,16 @@ You can run the Mule runtime standalone by starting the Mule service inside `$MU
 
 * On Windows environments:
 +
+** Install Windows Wrapper Service:
```

Comment:
Is this WinSW (installed by mule.bat?)? If so it should be Windows service wrapper here and in line 56. If it's something else, what is it? Also, it's no different than just starting Mule, so are we just warning people that part of Mule service is installing Windows service wrapper?

Reply:
Hello, Before starting Mule as a Windows Service, it must be installed by the bat. https://docs.mulesoft.com/mule-runtime/3.7/configuring-mule-as-a-windows-service has the correct procedure.